bin/ttab

```diff
@@ -417,6 +442,11 @@ if [[ -n $tabTitle ]]; then # custom tab title specified
   fi
 fi
 
+if (( gnomeTerminal )); then
+  script="$terminalApp $CMD_NEWTAB_1 $CMD_SWITCH_TO $CMD_TITLE -- $SHELL -c '$CMD_CUSTOM $CMD_KEEP_TAB_OPEN'"
+  echo $script
```

Comment:
The command should only be echoed if either `--dry-run` or `--print-script` are specified (which is covered below).

Reply:
Removed echo.